.github/workflows/ci.yml

```
@@ -100,6 +100,29 @@ jobs:
       - name: Check unused dependencies
         run: cargo machete
 
+  check-msrv:
+    name: Check MSRV
+    runs-on: ubuntu-latest
+    steps:
+
+      - uses: actions/checkout@v4.2.2
+
+      - name: Install Rust 1.74.0 toolchain
+        uses: actions-rs/toolchain@v1.0.7
+        with:
+          profile: minimal
+          toolchain: 1.74.0
```

I missed this at first so maybe just to make it obvious:

```suggestion
          # Check that tests pass using our MSRV:
          toolchain: 1.74.0
```

---

updated